README.md

```diff
@@ -4,42 +4,42 @@ wpantund, Userspace WPAN Network Daemon
 `wpantund` is a user-space network interface driver/daemon that
 provides a native IPv6 network interface to a low-power wireless
 **Network Co-Processor** (or *NCP*). It was written and developed by
-Nest Labs to make supporting [Thread](http://threadgroup.org)
-connectivity on unix-like operating systems more straightforward.
+Nest to support [Thread](http://threadgroup.org)
+connectivity on uUnix-like operating systems and make the support more straightforward.
 
-`wpantund` is designed to marshall all access to the NCP, ensuring
-that it always remains in a consistent, well-defined state.
+`wpantund` is designed to marshall all access to the NCP ensuring
+that it always remains in a consistent and well-defined state.
 
 This is not an official Google product.
 
-## Feature and Architecture Summary ##
+## Feature and architecture summary ##
 
 `wpantund` provides:
 
- *  ...a native IPv6 interface to an NCP.
- *  ...a command line interface (`wpanctl`) for managing and
+ *  Native IPv6 interface to an NCP.
+ *  Command line interface (`wpanctl`) for managing and
     configuring the NCP.
- *  ...a DBus API for managing and configuring the NCP.
- *  ...a way to reliably manage the power state of the NCP.
- *  ...a uniform mechanism for handling NCP firmware updates.
+ *  DBus API for managing and configuring the NCP.
+ *  Method to reliably manage the power state of the NCP.
+ *  Uniform mechanism for handling NCP firmware updates.
```

Comment:
Feel free to remove the `...` part if it seems to gimmicky, but I think the items in the list read awkwardly without at least an "A" at the beginning of each. Thoughts?